### PR TITLE
Added terratest scheduler

### PR DESCRIPTION
**Pull request by umarali-nagoor**
Test Results

Screenshot 2021-07-01 at 11 31 29 AM

### CODE REVIEW

File: examples/cluster-e2e/logging.tf [hunk 1]
```diff
@@ -0,0 +1,28 @@
+#####################################################
+# LOGDNA Configuration
+# Copyright 2021 IBM
+#####################################################
+
+module "logdna_instance" {
+  source = "terraform-ibm-modules/observability/ibm//modules/logging-logdna"
```
Comment (hkantare) [inline examples/cluster-e2e/logging.tf:7]: still source is reffering from registry

File: examples/cluster-e2e/monitoring.tf [hunk 2]
```diff
@@ -0,0 +1,28 @@
+#####################################################
+# SYSDIG Configuration
+# Copyright 2021 IBM
+#####################################################
+
+module "sysdig_instance" {
+  source = "terraform-ibm-modules/observability/ibm//modules/monitoring-sysdig"
```
Comment (hkantare) [inline examples/cluster-e2e/monitoring.tf:7]: same as above

File: examples/cluster-e2e/logging.tf (hunk 1 above)
Comment (ibmtfprovider) [reply examples/cluster-e2e/logging.tf:7]: To provision a logdna instance we need to source to registry level

File: examples/cluster-e2e/monitoring.tf (hunk 2 above)
Comment (ibmtfprovider) [reply examples/cluster-e2e/monitoring.tf:7]: To provision a sysdig instance we need to source to registry level

**Pull request merged**